src/main/java/joinery/DataFrame.java

```diff
@@ -246,22 +247,20 @@ private DataFrame(final Index index, final Index columns, final BlockManager<V>
      * > df.columns();
      * [value] }</pre>
      *
-     * @param columns the new column names
+     * @param column the new column name
      * @return the data frame with the columns added
      */
-    public DataFrame<V> add(final Object ... columns) {
-        for (final Object column : columns) {
+    public DataFrame<V> add(final Object column) {
```

Comment:
Why is an API change required?  Is this related to resolving the data/time format issues?  If not, please separate these changes into another pull request indicating the feature you would like implemented.